src/test/java/org/broadinstitute/hellbender/utils/variant/GATKVariantContextUtilsUnitTest.java

```diff
@@ -93,6 +97,48 @@ public void testHomozygousAlleleList() throws Exception {
         Assert.assertEquals(alleles, Arrays.asList(T, T));
     }
 
+    final static private Locatable START_AT_1 = new SimpleInterval("1",1,1);
+    final static private Locatable START_AT_2 = new SimpleInterval("1",2,2);
+
+    @DataProvider
+    Object[][] provideDataForDetermineReferenceAllele() {
+        // {list of vcs' allele lists, list of vc starts, Locatable for this start, expected ref allele
+        return new Object[][] {
+                { Collections.emptyList(), Collections.emptyList(), null, null },
+                { Collections.emptyList(), Collections.emptyList(), START_AT_1, null },
+                { Arrays.asList(Arrays.asList(Aref)), Arrays.asList(1), START_AT_1, Aref },
```

Comment:
Can you use `Collections.singletonList` for the calls to `Arrays.asList` with only one element?

Reply:
done